src/v2i-hub/MessageLoggerPlugin/src/MessageLoggerPlugin.cpp

```diff
@@ -43,23 +34,6 @@ MessageLoggerPlugin::MessageLoggerPlugin(string name): PluginClient(name)
 	PLOG(logDEBUG) << "Exit MessageLoggerPlugin Constructor";
 }
 
-/**
- * Destructor
- */
-
-MessageLoggerPlugin::~MessageLoggerPlugin()
```

Comment:
Why did we remove this?

Reply:
ofstream has a destructor, so we don't need this to close the files. I had to do this with other plugins in the past.